### PR TITLE
typos

### PR DESCRIPTION
**Pull request by Daniel-Worrall**
# Summary

Couple redundant code removals.

Various typo fixups through spell-checker.

### CODE REVIEW

File: lib/discordrb/data/server.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ class Server
     include IDObject
     include ServerAttributes
 
-    # @return [String] the ID of the region the server is on (e.g. `amsterdam`).
+    # @return [String] the ID of the region the server is on (e.g. `Amsterdam`).
```
Comment (swarley) [inline lib/discordrb/data/server.rb]: `amsterdam` is actually correct in this context. `region_id` differs from the region name

File: lib/discordrb/bot.rb [hunk 2]
```diff
@@ -168,10 +168,7 @@ def servers
     #   The list of emoji the bot can use.
     #   @return [Array<Emoji>] the emoji available.
     def emoji(id = nil)
-      gateway_check
-      unavailable_servers_check
```
Comment (z64) [inline lib/discordrb/bot.rb]: This check needs to stay

Comment (Daniel-Worrall) [reply lib/discordrb/bot.rb]: This check is called in Bot#servers

Comment (z64) [reply lib/discordrb/bot.rb]: Ah, right. NP then

**Comment by Daniel-Worrall** [comment]
Merged in the [new repo](https://github.com/shardlab/discordrb) in cc2900b582f4a7141318833fa2f5e7d754e1b6f4

**Pull request closed without merging**